autopush/main.py

```diff
@@ -500,8 +501,10 @@ def endpoint_main(sysargs=None, use_files=True):
 
     # Endpoint HTTP router
     site = cyclone.web.Application([
-        (r"/push/(?:(v\d+)\/)?([^\/]+)", EndpointHandler,
+        (r"/push/(?:(?P<api_ver>v\d+)\/)?(?P<token>[^\/]+)", EndpointHandler,
          dict(ap_settings=settings)),
+        (r"/spush/(?:(?P<api_ver>v\d+)\/)?(?P<token>[^\/]+)",
```

Comment:
Do we want to invalidate existing Simple `/push` endpoints, or will your Web Push refactor handle them?